vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -20,12 +20,20 @@ import (
 	"flag"
 
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/recommender/model"
+	"k8s.io/klog"
 )
 
 var (
 	safetyMarginFraction = flag.Float64("recommendation-margin-fraction", 0.15, `Fraction of usage added as the safety margin to the recommended request`)
 	podMinCPUMillicores  = flag.Float64("pod-recommendation-min-cpu-millicores", 25, `Minimum CPU recommendation for a pod`)
 	podMinMemoryMb       = flag.Float64("pod-recommendation-min-memory-mb", 250, `Minimum memory recommendation for a pod`)
+
+	targetCPUPercentile        = flag.Float64("recommendation-target-cpu-percentile", 0.9, `Percentile of cpu usage histogram for target recommendation for a pod`)
```

Comment:
nit: since VPA recommends per container, it's more accurate to say "recommendation for a container" here and below. (The podMin... flags above do target a whole pod, hence the "for a pod" in descriptions above)